docs/_pages/how-to-guide.md

```diff
@@ -7,15 +7,28 @@ toc: true
 # Prerequisites
 * [terraform >= 0.12](https://www.terraform.io/)
   - credentials configured for an AWS account
+  - note, some modules are pinned to 0.14 It is advised to use this version for now for best compatability. 
```

Comment:
Let's be more specific. For root modules, we always recommend a strict pin on the terraform-core version. This pin is to prevent inadvertent upgrades of tfstate by different people working the same project. The pin is in the file `configs/versions.tf`. To use another version, just update the version in the file.

All modules used ought to be compatible from 0.13 and later, when support for module-level `for_each` was added. It is expected that everything is compatible to 1.x.

Reply:
Ahhhhhh I see, that's where the error came from! Will update.